### DISCUSSION TITLE
anyone have python for mips

### DISCUSSION THREAD

**Discussion by endertable** [gtxaspec/wz_mini_hacks#452] [Development]
@baudev I see you were interested in compiling programs for the cam.  Did you ever get your toolchain to work?  I am asking to what the level of difficulty would be to get python compiled for the cam.  I get a compilation on mine but it does not want to run.  I get the 

'-sh: ./python: not found'

error.  I am sure it is an interpretor error, but I also can't get it to '--static' compile and run either.  Wondering if yours would compile it?  I tried the source at:

http://sources.buildroot.net/python3/Python-3.10.1.tar.xz

Thanks for any info..

**Comment by baudev** [comment]
Hi @endertable 

Yes, I managed to use the compilation chain corresponding to the one described in the following doc: https://github.com/gtxaspec/wz_mini_hacks/blob/master/documentation/developer_environment.md#buildroot-second

I compiled some programs like `inotify-tools` or `motion`
Sorry I didn't explore either the difficulty level to get Python or Node.js for the camera

Nobody has looked into this (even on the atomcam repo)?
